ethicalads-theme/templates/ea/advertisers.html

```diff
@@ -45,8 +45,8 @@ <h1 class="display-3 font-weight-light" data-aos="fade-left">Reach millions of e
     <div class="row justify-content-center">
 
       <div class="col-md-6 mb-3">
-        <h2 class="font-weight-bold mb-8">Get in front of developers</h2>
-        <p class="font-size-lg text-gray-800">When you run a campaign with us, your ad only appears on developer-focused websites. This drives your business results while supporting the developers creating these sites.</p>
+        <h2 class="font-weight-bold mb-8">Reach curious, engaged developers</h2>
+        <p class="font-size-lg text-gray-800">When you run a campaign with us, your ads are put front and center on vital developer resources like documentation and webtools, so your solution reaches real developers.</p>
```

Comment:
This is a great improvement, but think `front and center` and `webtools` are not the right term. Are we trying to focus on the fact that they are doing active dev, instead of in their off hours?

```suggestion
        <p class="font-size-lg text-gray-800">When you run a campaign with us, your ads are displayed on developer documentation and tools, so your solution reaches developers actively looking to solve a problem.</p>
```